lib/src/components/NxTextInput/__tests__/NxTextInput.test.tsx

```diff
@@ -61,7 +55,13 @@ describe('NxTextInput', function() {
   });
 
   it('sets the nx-tooltip--validation-error class on the tooltip', function() {
-    expect(getShallowComponent({ validationErrors: 'foo' })).toHaveClassName('nx-tooltip--validation-error');
+    expect(getShallowComponent()).toHaveClassName('nx-tooltip--validation-error');
```

Comment:
This seems like it reverses the meaning of the test.  Also, either way the test seems inadequate because it doesn't test the case when the `nx-tooltip--validation-error` class _shouldn't_ be present

Reply:
> This seems like it reverses the meaning of the test. Also, either way the test seems inadequate because it doesn't test the case when the `nx-tooltip--validation-error` class _shouldn't_ be present

Interestingly the code as currently established (i.e. before I made any changes) shows the `nx-tooltip--validation-error` class unconditionally. 

I don't remember changing this test (though obviously I did according to the log) but currently there is no scenario in which that class is not present. I guess that goes back to the whole discussion of whether the tooltip should be displayed conditionally or not

Reply:
> currently there is no scenario in which that class is not present.

That sounds like a bug.

Reply:
> > currently there is no scenario in which that class is not present.
> 
> That sounds like a bug.

Maybe? If a tooltip is not open, does it matter what CSS class is on it?

Reply:
> Maybe? If a tooltip is not open, does it matter what CSS class is on it?

No it doesn't, and I hadn't looked closely enough before to understand that.  This tooltip _is_ always a validation error tooltip.  It just isn't visible if there is in fact no validation error.  I think the test as-is in this PR is fine then.